cypress/integration/Case07_Send_money.js

```diff
@@ -15,16 +15,41 @@ describe('Test case 7: Ability to send money', () => {
       LoginPage.mnemonicsInput.type(mnemonic)
       LoginPage.recoverWalletButton.click()
       LoginPage.yayButton.click()
-      HomePage.claimButton.click().invoke('text').then(text => {
-        cy.log(text)
-        if (text == 'Queue') {
-          const urlRequest = Cypress.env('REACT_APP_SERVER_URL')
-          const bodyPass = Cypress.env('CYPRESS_GUNDB_PASSWORD')
-          cy.request('POST', urlRequest + '/admin/queue', [{ password: bodyPass, allow: 1 }])
-        }
+      HomePage.claimButton.click()
+      cy.wait(2000) // wait for a pop-up to appear confirming the decision in the queue
+      cy.get('#root')
+        .find('[role="button"]')
+        .its('length')
+        .then(res => {
+          cy.log(res)
+          if (res == 4) {
+            cy.get('[role="button"]')
```

Comment:
Aren't you generally pre-define any buttons/other UI controls via PageObject to re-use them and decrease tests cases code amount ?